apps/data-service-catalog/components/data-service-form/index.tsx

```diff
@@ -68,14 +69,18 @@ const DataServiceForm = ({ initialValues, submitType, searchEnv, referenceData,
   return (
     <Formik
       initialValues={initialValues as DataService}
+      validationSchema={dataServiceValidationSchema}
```

Comment:
Er det fremdeles mulig å lagre uten validering/påkrevde felter?

Reply:
Niet. Først bør vi sikkert prate om det hvordan det skal implementeres, regner med at det blir noe ala det Jeff la til i begrep, men tenker det fortjener en avsjekk med Kurt Stian og Kjersti først.

TLDR: Tar det i egen PR

Reply:
I begrep skal det være slikt:

1. Validering utføres ved lagring eller ved onBlur og onChange etter brukeren har forsøkt å lagre begrepet
2. Validering av obligatoriske felt er av by default
3. Validering av obligatoriske felt (krav til publisering) må brukeren huke av før lagring
4. Validering skjer på detaljsiden for å sjekke om brukeren for lov til å publisere begrepet

Reply:
#995